morpheushelper/cogs/color_picker.py

```diff
@@ -0,0 +1,32 @@
+from PyDrocsid.translations import translations
+from discord import Embed, File, Colour
+from discord.ext import commands
+from discord.ext.commands import Cog, Bot
+import re
+from PIL import ImageColor, Image
+import io
+
+
+class ColorPickerCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(name="colorpicker", aliases=["cp", "color"])
+    async def colorpicker(self, ctx, color_code: str):
+        if not (color_code_re := re.match(r'^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$', color_code)):
+            embed: Embed = Embed(title=translations.f_error_parse_color(color_code),
+                                 description=translations.error_parse_color_example)
+            await ctx.send(embed=embed)  # TODO
```

Comment:
if there is a todo you should resolve it before it get merged otherwise feel free to remove the comment